Source/Editor/Windows/Assets/PrefabWindow.Hierarchy.cs

```diff
@@ -366,6 +366,9 @@ public void Spawn(Actor actor)
                 actor.Transform = parentActor.Transform;
```

Comment:
You could simplify this code by just changing this line to `actor.LocalTransform = Transform.Identity;` and removing the 3 times you wrote. No need to set the transform 2 times.

Reply:
You could replace this line of code with that identity code, or since you added it below, you can remove this line, no need to set the transform 2 times.